### PR TITLE
bext: disable dark mode and enable screenshots for certain stories

### PR DESCRIPTION
**Pull request by taras-yemets**
Closes [#30341](https://github.com/sourcegraph/sourcegraph/issues/30341)

## Test plan
Run `yarn storybook:browser` and check the updated stories.

### CODE REVIEW

**Review by slimsag — APPROVED** [review]

File: client/browser/src/browser-extension/after-install-page/AfterInstallPageContent.story.tsx [hunk 1]
```diff
@@ -11,7 +11,6 @@ const config: Meta = {
     title: 'browser/AfterInstallPage',
     parameters: {
         chromatic: {
-            enableDarkMode: true,
```
Comment (erzhtor) [inline client/browser/src/browser-extension/after-install-page/AfterInstallPageContent.story.tsx:14]: I think we still want to test this on the dark theme. WDYT?

File: client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx [hunk 2]
```diff
@@ -46,49 +46,90 @@ const OptionsPageWrapper: React.FunctionComponent<Partial<OptionsPageProps>> = p
...
 )
 
-OnPrivateRepository.storyName = 'On private repository'
-
-export const OnSourcegraphCloud: Story = () => (
+const OnSourcegraphCloud: Story = () => (
     <OptionsPageWrapper requestPermissionsHandler={requestPermissionsHandler} showSourcegraphCloudAlert={true} />
 )
+
+const WithAdvancedSettings: Story = () => {
```
Comment (erzhtor) [inline client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx:76]: I think we can ignore all other stories in the storybook display and include only a single "AllOptionsPages" to keep it clean:
```
const config: Meta  = {
  ...
  includeStories: ['AllOptionsPages']
}
```
https://storybook.js.org/docs/react/api/csf#non-story-exports

File: client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx [hunk 3]
```diff
@@ -46,49 +46,90 @@ const OptionsPageWrapper: React.FunctionComponent<Partial<OptionsPageProps>> = p
...
+    return (
+        <OptionsPageWrapper
+            initialShowAdvancedSettings={true}
+            optionFlags={optionFlagValues}
+            onChangeOptionFlag={setOptionFlag}
+        />
+    )
+}
+
+export const AllOptionsPages: Story = () => (
```
Comment (erzhtor) [inline client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx:94]: Looks great! 👍

**Review by erzhtor — APPROVED** [review]
Awesome job! 💪 

One minor thing, WDYT if we [update the order of variants](https://www.chromatic.com/component?appId=5f0f381c0e50750022dc6bf7&name=browser%2FOptions%2FOptionsPage&buildNumber=37732&componentInspectorKey=620d1338e45f59003a41a19f-1200-interactive-true&historyLengthAtIndex=5&distanceToMoveBack=-1) to make short ones on the first row? (To move "With advanced settings" as 3rd item I guess)

File: client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx (hunk 2 above)
Comment (taras-yemets) [reply client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx:76]: `AllOptionsPages` is the only exported story in this file and so it's the only item in the side navigation.

browser - Options - OptionsPage - All Options Pages ⋅ Storybook 2022-02-17 10-52-44
Is it what you mean?

Comment (erzhtor) [reply client/browser/src/browser-extension/options-menu/OptionsPage.story.tsx:76]: Oh, sorry. I misunderstood.👌

**Pull request merged**